docs/atlascli/command/atlas-backups-exports-buckets-create.txt

```diff
@@ -0,0 +1,95 @@
+.. _atlas-backups-exports-buckets-create:
+
+====================================
+atlas backups exports buckets create
+====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Create new export bucket.
+
+Syntax
+------
+
+.. code-block::
+
+   atlas backups exports buckets create <bucketName> [options]
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - bucketName
+     - string
+     - false
+     - Name of the bucket that the provided role ID is authorized to access.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --cloudProvider
+     - string
+     - true
+     - Name of the provider of the cloud service where Atlas can access the S3 bucket. Atlas only supports AWS.
+   * - -h, --help
+     - 
+     - false
+     - help for create
+   * - --iamRoleId
+     - string
+     - true
+     - Unique identifier of the role that Atlas can use to access the bucket.
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file.
+   * - --projectId
+     - string
+     - false
+     - Project ID to use. Overrides the settings in the configuration file or environment variable.
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Profile to use from your configuration file.
+
+Examples
+--------
+
+.. code-block::
+
+   Create new export bucket:
```

Comment:
Suggestion: aside from this text change, I would change the name of the bucket to something more generic not including a name ("test-bucket" works)
```suggestion
   The following command creates an export destination for Atlas backups using the existing AWS S3 bucket named  michal-test-bucket-2
```

Reply:
Yeah, that's a bad bucket name, somehow I didn't think about it. 😅 Fixed!